files/srv/shop/lib/tasks/password.rake

```diff
@@ -0,0 +1,9 @@
+namespace :workarea do
+  namespace :admin do
+    task password: :environment do
+      password = SecureRandom.base64
```

Comment:
fun fact: `SecureRandom.base64` generates a String with both uppercase and lowercase characters, numbers, and symbols, and it's always longer than our minimum password length. Pretty good replacement for `SecureRandom.hex + "aA1"`!